cvmfs/cvmfs_server

```diff
@@ -2026,6 +2027,38 @@ Alias /cvmfs/${name} ${storage_dir}
 EOF
 }
 
+create_apache_config_for_global_info() {
+  local conf_file="cvmfs.info.conf"
+
+  ! has_apache_config_file $conf_file || return 0
+
+  # TODO(rmeusel): make this a bit more flexible
+  local storage_dir="${DEFAULT_LOCAL_STORAGE}/info"
+
+  # TODO(rmeusel): fine-tune this configuration file
+  create_apache_config_file $conf_file << EOF
+# Created by cvmfs_server.  Don't touch.
+
+KeepAlive On
+Alias /cvmfs/info ${storage_dir}
+<Directory "${storage_dir}">
+    Options -MultiViews
+    AllowOverride Limit
+    $(get_compatible_apache_allow_from_all_config)
+
+    EnableMMAP Off
+    EnableSendFile Off
+
+    FileETag None
+
+    ExpiresActive On
+    ExpiresDefault "access plus 3 days"
+    ExpiresByType text/html "access plus 5 minutes"
+    ExpiresByType application/x-cvmfs "access plus 2 minutes"
+</Directory>
+EOF
+}
+
```

Comment:
Well a lot of it would be in common anyway.  The only parts that would be different would be the expiration parameters, but cvmfs-servermon has to override them anyway (with Cache-control: max-age header in the request, as discussed in the [ticket](https://sft.its.cern.ch/jira/browse/CVM-860?focusedCommentId=68835&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-68835)) at least when reading per-repo .cvmfs* files so it might as well do the same here.  I think it would be easiest/best to just leave the apache config the same.